materialize-cratedb/Dockerfile

```diff
@@ -0,0 +1,43 @@
+ARG BASE_IMAGE=ghcr.io/estuary/base-image:v1
```

Comment:
We need to add the connector to the CI process so it will get built. See https://github.com/estuary/connectors/commit/8bce00f175db9824b658d2644435f6366c6608ff for an example of doing this. 

For this case, it will need to be added to the list in the `build_connectors` matrix, but probably not to the `Materialization connector ${{ matrix.connector }} integration tests` matrix, since it doesn't look like those are fully working yet.